tests/subsys/tracing/testcase.yaml

```diff
@@ -0,0 +1,12 @@
+tests:
+  tracing.transport:
+    tags: tracing_disabled
+  tracing.transport.uart:
+    extra_args: CONF_FILE="prj_uart.conf"
+    filter: CONFIG_UART_CONSOLE and not CONFIG_QEMU_TARGET
+    tags: tracing_uart
+  tracing.transport.usb:
+    extra_args: CONF_FILE="prj_usb.conf"
+    filter: CONFIG_UART_CONSOLE and not CONFIG_QEMU_TARGET
```

Comment:
why is qemu being excluded here? when does this test get run? I see it now only building for all tests where tracing is enabled and only runs on tests that do not enable tracing, so what is the point?

Reply:
this tests worked for phisical boards,  and Let me have a double check if qemu support tracing output....

Reply:
I have fixed this and enabled the qemu platform.